Tasks/Loktionova/Task3/task3.py

```diff
@@ -0,0 +1,45 @@
+with open ('text.txt', 'r', encoding='utf-8') as file, open('new_text.txt', 'w') as new:
+
+    line = file.read().replace('\n', ' ')
```

Comment:
...and reading the whole file in memory right on the next line

it's not advised approach, since the file can be very big, -10 points for the not optimal strategy